src/xmrEngine.js

```diff
@@ -807,7 +813,7 @@ class MoneroEngine {
         sendParams
       }
     }
-
+    this.log.warn(`makeSpend result ${JSON.stringify(result)}`)
```

Comment:
Lets not log the entire result object. At minimum create a cleaner that only returns the keys we want to show.